manual/cypher/cypher-docs/src/docs/graphgists/sql/cypher-vs-sql.asciidoc

```diff
@@ -143,11 +143,11 @@ CREATE
   (AndyW)    -[:DIRECTED]->    (TheMatrix),
   (LanaW)    -[:DIRECTED]->    (TheMatrix),
   (JoelS)    -[:PRODUCED]->    (TheMatrix)
-CREATE (TheDevilsAdvocate:Movie {title:"The Devil's Advocate", released:1997, tagline: 'Evil has its winning ways'})
+CREATE (TheDevilsAdvocate:Movie {title: "The Devil's Advocate", released: 1997, tagline: 'Evil has its winning ways'})
```

Comment:
That's another one for the style guide: when a string literal contains a `'`, should one use `"` for the string literal as a whole, or should one resort to escpaing, `\'`?